sdk/storage/storage-blob/review/storage-blob.api.md

```diff
@@ -3195,6 +3195,12 @@ export interface StaticWebsite {
     indexDocument?: string;
 }
 
+// @public
+export enum StorageBlobAudience {
+    DiskComputeOAuthScopes = "https://disk.compute.azure.com/.default",
```

Comment:
Is it disk.compute.azure.com or disk.azure.com? Used to read spec [https://microsoft-my.sharepoint.com/:w:/p/rerdmann/EQ9glSbPNtdDu2-BrazUX0oBUpdPu3OnIK_D9uGTlGZtQQ?e=k08rDC](https://microsoft-my.sharepoint.com/:w:/p/rerdmann/EQ9glSbPNtdDu2-BrazUX0oBUpdPu3OnIK_D9uGTlGZtQQ?e=k08rDC) where disk.azure.com is mentioned, worth to double check.